src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataParent.java

```diff
@@ -351,6 +353,9 @@ public class CaseDataParent implements MappableObject {
     @JsonFormat(shape = JsonFormat.Shape.STRING)
     private final BigDecimal ccjJudgmentAmountInterestToDate;
 
+    @JsonUnwrapped
+    private final CaseDataCui caseDataFromCui;
```

Comment:
Perhaps better if it could be called CaseDataLip?

Reply:
Solve, please review it again <3